python/makeAlpacaEvents.py

```diff
@@ -0,0 +1,146 @@
+import os,sys,getopt
+import numpy,math
+import ROOT
+
+hGeV = 6.58211928*pow(10.,-16)* pow(10.,-9) # no units or it messes up!!
+c_light = 2.99792458e+10
+mres="1.0d0"
+gax="1d-7"
+nev="100"
+
+def AlpacaFormatting(s):
+    s=str(s)
+    if s.find('d')==-1:
+        s = numpy.format_float_scientific(float(s))
+        s= s.replace('e','d')
+        s= s.replace('+','')
+    return s
+
+#determine the path for online usage it can be /eos/../ship/data/.. BUT don't know what we shall do in local usage??
+ 
+def Ctau(mres,gax):
+    return c_light*hGeV*65.*math.pi/((float(gax)*float(gax))*(float(mres)*float(mres)*float(mres)))
+
+def Decaylength(e,p,ctau):
+    beta=p/e
+    gamma=e/math.sqrt(e*e-p*p)
+    return beta*gamma*ctau
+
+def Decayweight(Lmin,Lmax,Decaylength,LS):
+    return math.exp(-LS/Decaylength)*((Lmax-Lmin)/Decaylength)
+
+def inputWrite(mres,gax,nev,Lmin,Lmax):#need to apply Rmin & Rmax as well..
+    mres    = AlpacaFormatting(mres)
+    gax     = AlpacaFormatting(gax)
+    L   = AlpacaFormatting(float(Lmax)-float(Lmin))
+    Lmin    = AlpacaFormatting(Lmin)
+    print Lmin,Lmax,L
+    f=open("input.DAT","w")
```

Comment:
Would suggest using a contextmanager instead:
```python
with open("...", "w") as f:
    f.write(...)
    ...
```
Closing the file is then automatic (even if there is a crash or something else), so does not have to be done manually.

Reply:
OK.